frontend/src/components/SideBar/sideBar.less

```diff
@@ -47,6 +47,9 @@
 
 @media only screen and (max-width: 600px) {
   #side-bar {
-    display: none;
+    position: absolute;
+    margin-left: 0;
+    width: 70%;
+    z-index: 2;
```

Comment:
This makes it always lay on top and makes the rest of the application impossible to use.
![](https://folk.ntnu.no/johannkv/img/eeP0FHOpXk.png)

Reply:
Whoops, yes this should've been in the sidebar pr